File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0801U00GrdOCS0802ListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class OCS0801U00GrdOCS0802ListItemInfo
    {
        private String _patStatus;
        private String _patStatusCode;
        private String _patStatusName;
        private String _ment;
        private String _seq;
        private String _contKey;
        private String _rowSate;

        public String PatStatus
        {
            get { return this._patStatus; }
            set { this._patStatus = value; }
        }

        public String PatStatusCode
        {
            get { return this._patStatusCode; }
            set { this._patStatusCode = value; }
        }

        public String PatStatusName
        {
            get { return this._patStatusName; }
            set { this._patStatusName = value; }
        }

        public String Ment
        {
            get { return this._ment; }
            set { this._ment = value; }
        }

        public String Seq
        {
            get { return this._seq; }
            set { this._seq = value; }
        }

        public String ContKey
        {
            get { return this._contKey; }
            set { this._contKey = value; }
        }

        public String RowSate
        {
            get { return this._rowSate; }
            set { this._rowSate = value; }
        }

        public OCS0801U00GrdOCS0802ListItemInfo() { }

        public OCS0801U00GrdOCS0802ListItemInfo(String patStatus, String patStatusCode, String patStatusName, String ment, String seq, String contKey, String rowSate)
        {
            this._patStatus = patStatus;
            this._patStatusCode = patStatusCode;
            this._patStatusName = patStatusName;
            this._ment = ment;
            this._seq = seq;
            this._contKey = contKey;
            this._rowSate = rowSate;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCS0801U00GrdOCS0802ListItemInfo.java
package nta.med.data.model.ihis.ocso;

public class OCS0801U00GrdOCS0802ListItemInfo {
	private String patStatus;
	private String patStatusCode;
	private String patStatusName;
	private String ment;
	private Double seq;
	private String contKey;
	public OCS0801U00GrdOCS0802ListItemInfo(String patStatus,
			String patStatusCode, String patStatusName, String ment,
			Double seq, String contKey) {
		super();
		this.patStatus = patStatus;
		this.patStatusCode = patStatusCode;
		this.patStatusName = patStatusName;
		this.ment = ment;
		this.seq = seq;
		this.contKey = contKey;
	}
	public String getPatStatus() {
		return patStatus;
	}
	public void setPatStatus(String patStatus) {
		this.patStatus = patStatus;
	}
	public String getPatStatusCode() {
		return patStatusCode;
	}
	public void setPatStatusCode(String patStatusCode) {
		this.patStatusCode = patStatusCode;
	}
	public String getPatStatusName() {
		return patStatusName;
	}
	public void setPatStatusName(String patStatusName) {
		this.patStatusName = patStatusName;
	}
	public String getMent() {
		return ment;
	}
	public void setMent(String ment) {
		this.ment = ment;
	}
	public Double getSeq() {
		return seq;
	}
	public void setSeq(Double seq) {
		this.seq = seq;
	}
	public String getcontKey() {
		return contKey;
	}
	public void setcontKey(String contKey) {
		this.contKey = contKey;
	}


}